pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -57,29 +64,26 @@ let
       postFixup = (attrs.postFixup or "") + lib.optionalString (stdenv.isLinux) ''
         (
           cd $out/clion
-          # bundled cmake does not find libc
-          rm -rf bin/cmake/linux
-          ln -s ${cmake} bin/cmake/linux
-          # bundled gdb does not find libcrypto 10
-          rm -rf bin/gdb/linux
-          ln -s ${gdb} bin/gdb/linux
+          ls -d $PWD/bin/gdb/linux/x64/lib/python3.8/lib-dynload/* |
+          xargs patchelf \
+            --replace-needed libssl.so.10 libssl.so \
+            --replace-needed libcrypto.so.10 libcrypto.so
```

Comment:
can you add a comment back in explaining why you are doing this?

Reply:
I think its because they add a couple of patches so some integration works better? I don't remember at all

Reply:
that's the reason why I would like a comment there :D

Reply:
done